Library/Formula/exploitdb.rb

```diff
@@ -0,0 +1,23 @@
+class Exploitdb < Formula
+  desc "The official Exploit Database"
+  homepage "https://www.exploit-db.com/"
+
+  url "https://github.com/offensive-security/exploit-database.git",
+    :tag => "2016-03-21",
+    :revision => "e67a88eeffbc511d3e5e10bfc26639e6fce16576"
+
+  head "https://github.com/offensive-security/exploit-database.git"
+
+  def install
+    inreplace "searchsploit", "gitpath=\"/usr/share/exploitdb\"", "gitpath=\"#{share}\""
+    bin.install "searchsploit"
+
+    share.install "files.csv"
+    share.install "platforms"
+    share.install ".git"
```

Comment:
I don’t think it works correctly:
```
$ searchsploit --update
error: pathspec '.' did not match any file(s) known to git.
From gh:Homebrew/homebrew
 * branch            master     -> FETCH_HEAD
Already up-to-date.
[*] Update finished.
```

(`gh:Homebrew/homebrew` is my Homebrew remote)